codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ConstrainedMapGenerator.kt

```diff
@@ -149,6 +154,7 @@ class ConstrainedMapGenerator(
                     type Unconstrained = #{UnconstrainedSymbol};
                 }
                 """,
+                *codegenScope,
```

Comment:
Maybe is an artefact of the diff, but are you sure you need `codegenScope` here?